src/typings.d.ts

```diff
@@ -1 +1 @@
-declare module 'iframe-phone';
\ No newline at end of file
+declare module "iframe-phone";
```

Comment:
I don't think this file is needed anymore since the api hides the usage of `iframe-phone`.